src/NHibernate.Test/Tools/hbm2ddl/SchemaMetadataUpdaterTest/SchemaMetadataUpdaterFixture.cs

```diff
@@ -140,7 +147,7 @@ public void CheckForExcessReservedWordsHardCodedInDialect()
 			}
 
 			// Don't fail incase the driver returns nothing.
-			//Assert.That(sf.Dialect.Keywords, Is.EquivalentTo(reservedDb));
+			// This is an info-only test.
```

Comment:
You may issue an `Assert.Warn` instead. Since a recent PR it will not cause the build to fail.
`Assume.That` would be slightly better, but it currently causes a failure, excepted in a PR where I have started using it and [fixed it](https://github.com/nhibernate/nhibernate-core/pull/627/files#diff-628b6f12bc5afe5f072a2e2f1b01f576).